test/recipes/30-test_evp_data/evpkdf_pbkdf2.txt

```diff
@@ -235,6 +236,7 @@ Ctrl.salt = salt:salt
 Ctrl.iter = iter:4096
 Ctrl.digest = digest:sha1
 Result = KDF_CTRL_ERROR
+Reason = invalid salt length
```

Comment:
Was this the intended result??

Reply:
That's what the code is raising as an error.  It's correct, the salt is too short.
I don't think this is the proper place to "fix" this, a new issue/PR would be.

Reply:
Issue raised: #25136

Reply:
It was correct at the time it was written since the code for the ems_check was done first.. It would have been detected if we has added the reason before adding the salt len check

Reply:
Moving targets are like this...